cumulusci/tasks/salesforce/ActivateProcessBuilderProcesses.py

```diff
@@ -0,0 +1,36 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class ActivateProcessBuilderProcesses(BaseSalesforceApiTask):
+    """
+    Create a Salesforce Activation Process Builder which takes in a list as options and
+    runs flows only on those listed DeveloperName's.
+    """
+
```

Comment:
Let's put a `task_options` in here so that we can show the options at the command line. (I'd suggest calling the option `developer_names` with an underscore for consistency).

Then, add an `_init_options()` method and you can use `process_list_arg()` from `cumulusci.core.utils` to turn the user input into a real Python list. There's an example in `testrunner.py`.

Reply:
Am I just passing in **kwargs into process_list_arg I' ve gotten the changes to work but only when I hard code the values, I'm going to push and submit to show.

Reply:
@jofsky The usual pattern is that `_init_options` should start by calling `super()._init_options(kwargs)` which will do some default processing including initializing `self.options` from `kwargs`. Then your code can make adjustments to `self.options`.